MeetU/MeetU/Views/Meetups/edit.html

```diff
@@ -0,0 +1,52 @@
+﻿<div ng-controller="MeetupEditController as editCreating">
```

Comment:
you got two edit.html in different dir. this one is incorrect. could you review your own code before you make a pull request?